### PR TITLE
modals: actually hide them to avoid links being clickable

### PR DESCRIPTION
**Pull request by muryoh**

### CODE REVIEW

File: src/components/modal/modal.module.css [hunk 1]
```diff
@@ -16,13 +16,13 @@
...
   box-shadow: var(--box-shadow-modal);
-  opacity: 0;
+  visibility: hidden;
   pointer-events: none;
   transform: translate(-50%, -50%);
   transition-property: opacity;
 }
 .open.center {
-  opacity: 1;
+  visibility: visible;
```
Comment (gqio) [inline src/components/modal/modal.module.css:25]: should not it display none instead ?

Comment (muryoh) [reply src/components/modal/modal.module.css:25]: Hm good question, dunno, will try

Comment (muryoh) [reply src/components/modal/modal.module.css:25]: "Should" is strong word, dunno if it's really better as CSS transitions don't apply with display: none switching
But I should have kept opacity for the same reason, thanks for pointing it out

Comment (gqio) [reply src/components/modal/modal.module.css:25]: I don't know tbh :) but display have different behaviour vs vibility

Comment (muryoh) [reply src/components/modal/modal.module.css:25]: ok, kept visibility for now for the aforementioned transition to work

**Pull request merged**